code/game/machinery/doors/door.dm

```diff
@@ -231,12 +231,12 @@ var/list/all_doors = list()
 	playsound(H.loc, 'sound/effects/horrorforce2.ogg', 80)
 	visible_message("<span class='danger'>\The [src]'s motors whine as several great tendrils begin trying to force it open!</span>")
 	if(do_after(H, src, 32))
-		open(1)
+		open()
```

Comment:
??

Reply:
open(1) causes a runtime, tries to pass value 1 as a ckey